controllers/bookingsController.js

```diff
@@ -52,12 +54,425 @@ class BookingsController extends BaseController {
+
+      @media screen and (max-width: 500px) {
+      }
+    </style>
+  </head>
+
+  <body
+    width="100%"
+    style="
+      margin: 0;
+      padding: 0 !important;
+      mso-line-height-rule: exactly;
+      background-color: #f1f1f1;
+    "
+  >
+    <center style="width: 100%; background-color: #f1f1f1">
+      <div
+        style="
+          display: none;
+          font-size: 1px;
+          max-height: 0px;
+          max-width: 0px;
+          opacity: 0;
+          overflow: hidden;
+          mso-hide: all;
+          font-family: sans-serif;
+        "
+      >
+        &zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;
+      </div>
+      <div style="max-width: 600px; margin: 0 auto" class="email-container">
+        <table
+          align="center"
+          role="presentation"
+          cellspacing="0"
+          cellpadding="0"
+          border="0"
+          width="100%"
+          style="margin: auto"
+        >
+          <tr>
+            <td
+              valign="top"
+              class="bg_white"
+              style="padding: 1em 2.5em 0 2.5em"
+            >
+              <table
+                role="presentation"
+                border="0"
+                cellpadding="0"
+                cellspacing="0"
+                width="100%"
+              >
+                <tr>
+                  <td class="logo" style="text-align: center">
+                    <h1><a href="#">Event Booking</a></h1>
+                  </td>
+                </tr>
+              </table>
+            </td>
+          </tr>
+          <tr></tr>
+          <tr>
+            <td
+              valign="middle"
+              class="hero bg_white"
+              style="padding: 2em 0 4em 0"
+            >
+              <table>
+                <tr>
+                  <td>
+                    <div
+                      class="text"
+                      style="padding: 0 2.5em; text-align: center"
+                    >
+                      <h2>Your Event booking is done.</h2>
+                      <h3>Contact Us for queries</h3>
+                      <p>
+                        <a
+                          href="mailto:eventlink@gmail.com"
+                          class="btn btn-primary"
+                          >eventlink@gmail.com</a
+                        >
+                      </p>
+                    </div>
+                  </td>
+                </tr>
+              </table>
+            </td>
+          </tr>
+        </table>
+      </div>
+    </center>
+  </body>
+</html>
+
+      `,
+      attachments: [
+        {
+          filename: "../done.png",
+          path: path.join(__dirname, "../done.png"),
+          contentType: "image/jpg",
+        },
+      ],
+    };
+
+    transporter.sendMail(mailOptions, function (error, info) {
```

Comment:
This nodemailer should be under async getSessionStatus(req, res) {}
I believe we want to send email after the user successfully make payment?